viewer/config.js

```diff
@@ -500,10 +500,65 @@ exports.configMap = function (section, dSection, d) {
   return map;
 };
 
+function loadCertData () {
+  exports.keyFileLocation = exports.get('keyFile');
+  exports.certFileLocation = exports.get('certFile');
+
+  // eslint-disable-next-line no-useless-catch
+  try {
+    exports.keyFileData = fs.readFileSync(exports.keyFileLocation);
+    exports.certFileData = fs.readFileSync(exports.certFileLocation);
+  } catch (err) {
+    throw err;
```

Comment:
why do you need this try/catch if you are just going to throw? Actually we need to test this where viewer can't read the cert file after the drop priv and make sure its not constantly erroring.

Reply:
I was using it to log consistent errors in the catch block, so I left it until we decided whether the we can assume that setServerToReloadCerts is called for each server that sets certs. Since you ok'd that assumption, I am going to remove the try catch and do all console logging in that function.

Reply:
Isn't any change to the cert files after the config loads testing the drop priv stuff?

Reply:
so if you do the monitor BEFORE the drop priv it might still fire after a change, but you might not be able to read the new file because it is owned by root and read only by root. In that case it might be better to restart. Not sure. I'm also not sure if file watching still works after drop priv, but you definitely wouldn't be able to open that file

The current test stuff doesn't test the drop priv, you would need to add
dropUser=nobody
dropGroup=nobody
make sure the test cert is own by root, chmod 600, and then start viewer as root. then try changing the file.
And then mac may be different than linux :)

Reply:
Also, in the end, I didn't like the assumption that setServerToReloadCerts would be called for every https server. I'm going a different route that we can discuss when I push the code.

Reply:
and to make sure I was clear, if there is no error it would be great to reset the cert without restarting. not everyone will have it so the cert isn't readable by nobody.